--- a/xm-uart/main.c	2021-11-06 21:52:52.000000000 +0300
+++ b/xm-uart/main.c	2022-12-11 19:22:03.322193742 +0300
@@ -132,7 +132,7 @@
   ctrl.c_lflag &= ~ECHO;   // disable echo
   tcsetattr(STDIN_FILENO, TCSANOW, &ctrl);
 
-  int uart = open("/dev/ttyAMA0", O_RDWR | O_NOCTTY);
+  int uart = open("/dev/ttyS1", O_RDWR | O_NOCTTY);
   if (uart == -1) {
     printf("Error no is : %d\n", errno);
     printf("Error description is : %s\n", strerror(errno));
